From 295c34a777e099a05222bb75ec511ff394d3d7a1 Mon Sep 17 00:00:00 2001
From: Wang Quanyang <quanyang.wang@windriver.com>
Date: Fri, 1 Jun 2018 22:30:40 -0400
Subject: [PATCH] ARM: imx: add missing map_io part for imx6ul

In commit 12b14f8cd8cf ("MLK-11300-05 ARM: imx: add pm support for imx6ul")
of git://git.freescale.com/imx/linux-imx.git imx_4.9.11_1.0.0_ga, the part
as below has been missed when this patch is picked up. So add it to fix the compile
warning as below:

arch/arm/mach-imx/mach-imx6ul.c:203:20: warning: 'imx6ul_map_io' defined but not used [-Wunused-function]
 static void __init imx6ul_map_io(void)
                     ^~~~~~~~~~~~~

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm/mach-imx/mach-imx6ul.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-imx/mach-imx6ul.c b/arch/arm/mach-imx/mach-imx6ul.c
index b5f0b6b..851034d 100644
--- a/arch/arm/mach-imx/mach-imx6ul.c
+++ b/arch/arm/mach-imx/mach-imx6ul.c
@@ -214,6 +214,7 @@ static void __init imx6ul_map_io(void)
 };
 
 DT_MACHINE_START(IMX6UL, "Freescale i.MX6 UltraLite (Device Tree)")
+	.map_io		= imx6ul_map_io,
 	.init_irq	= imx6ul_init_irq,
 	.init_machine	= imx6ul_init_machine,
 	.init_late	= imx6ul_init_late,
-- 
1.7.9.5

